css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 5em;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 5em;
```

Comment:
Oh no, this is a _**huge**_ movement. Instead, you should try to position the element from the edge closest to where it will actually be, which in this case is `bottom`

Reply:
You missed this. Please double-check that you have addressed all feedback before re-submitting

Reply:
I apologize. Will make the correction immediately.